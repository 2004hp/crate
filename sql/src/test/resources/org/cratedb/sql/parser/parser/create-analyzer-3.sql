create analyzer myAnalyzer extends otherAnalyzer with (
    token_filters with (
        "my_filter" with (
            bla='blubb'
        ),
        myOtherTokenFilter (
            "false"=true,
            one=1
        )
    )
)