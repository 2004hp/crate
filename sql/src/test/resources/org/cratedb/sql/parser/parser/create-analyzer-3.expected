org.cratedb.sql.parser.parser.CreateAnalyzerNode@59a6acb9
name: myanalyzer
statementType: CREATE ANALYZER
	extends: 	
	org.cratedb.sql.parser.parser.TableName@8e9abbb6
	otheranalyzer
	
	org.cratedb.sql.parser.parser.AnalyzerElements@54266750

	TokenFilters:
		
		org.cratedb.sql.parser.parser.TokenFilterList@2548e059

		[0]:		
		org.cratedb.sql.parser.parser.TokenFilterNode@2548e059
		my_filter
		Properties: 			
			org.cratedb.sql.parser.parser.GenericProperties@5381062e
			{bla=value: blubb
			type: CHAR(5) NOT NULL
			}
		[1]:		
		org.cratedb.sql.parser.parser.TokenFilterNode@3bbce0d6
		myothertokenfilter
		Properties: 			
			org.cratedb.sql.parser.parser.GenericProperties@6d4d203d
			{one=value: 1
			type: INTEGER NOT NULL
			, false=value: true
			type: BOOLEAN NOT NULL
			}
	CharFilters:
		
		org.cratedb.sql.parser.parser.CharFilterList@627fbcda
